dashboard/data/bmg_plate.py

```diff
@@ -260,18 +260,18 @@ def get_activation_inhibition_zscore_dict(
 
 def filter_low_quality_plates(
     df: pd.DataFrame, plate_array: np.ndarray, threshold: float = 0.5
-) -> tuple[pd.DataFrame, np.ndarray]:
+) -> tuple[pd.DataFrame, np.ndarray, int]:
     """
     Remove plates with Z factor lower than threshold
-    TODO Add alert in dash how many plates were deleted
 
     :param df: DataFrame with control values
     :param plate_array: array with plate values
     :param threshold: Z factor threshold value
-    :return: high quality plates
+    :return: high quality plates, number of deleted plates
     """
     quality_mask = df.z_factor > threshold
     quality_df = df[quality_mask]
     low_quality_ids = np.where(quality_mask == False)
+    print(low_quality_ids)
     quality_plates = np.delete(plate_array, low_quality_ids, axis=0)
-    return quality_df, quality_plates
+    return quality_df, quality_plates, len(low_quality_ids[0])
```

Comment:
isn't it better to just calculate the difference between the sizes of passed `df` and the returned `quality_df` outside the function instead of returning it from the filtering function? e.g.

```python
filtered_df, filtered_vals = filter_low_quality_plates(bmg_df, bmg_vals)
num_removed = bmg_df.shape[0] - filtered_df.shape[0]
```

Reply:
yeah that makes sense